enso-org/enso
# Expand parent directories of opened projects

### Pull Request Description
- https://github.com/enso-org/cloud-v2/issues/1499
  - Expand parent directories of opened projects when switching to directories

### Important Notes
None

### Testing Instructions
- Open a project on each backend (cloud and local) and test that it works for both
- Open a project and make sure it's expanded on a different category (team category and root, user category and root, add subdirectory of `enso/` to local favorites and open project)

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.~~
  - ~~If applicable, it is suggested to paste a link to a successful run of the Extra Tests.~~
